spec/forms/state_file/esign_declaration_form_spec.rb

```diff
@@ -235,4 +272,110 @@
     end
   end
 
+  describe "#validations with signature pin" do
+    let!(:intake) { create :state_file_md_intake, primary_esigned: "unfilled", primary_esigned_at: nil, spouse_esigned: "unfilled", filing_status: "married_filing_jointly", primary_signature_pin: "unfilled", spouse_signature_pin: "unfilled" }
+
+    context "when filing" do 
+      before do
+        allow(intake).to receive(:filing_status_single?).and_return(true)
+        allow(intake).to receive(:ask_for_signature_pin?).and_return(true)
+      end
+
+      it "pin cannot be 00000" do
```

Comment:
This scenario is tested in `signature_pin_validator_spec.rb` so I would leave it out of here. The "pin cannot be less than 5 digits" scenario isn't in that spec though, so moving that one over would be a good idea.